src/Replacements.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-zendframework-bridge for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-zendframework-bridge/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-zendframework-bridge/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\ZendFrameworkBridge;
+
+class Replacements
+{
+    /** @var string[] */
+    private $replacements;
+
+    public function __construct(array $additionalReplacements = [])
+    {
+        $this->replacements = array_merge(
+            require __DIR__ . '/../config/replacements.php',
+            $additionalReplacements
+        );
+    }
+    
+    /**
+     * @param string $value
+     * @return string
+     */
+    public function replace($value)
+    {
+        if (! is_string($value)) {
```

Comment:
why do we need that case here? we expecting that input parameter is string, and in PHP 7 we would add `string` typehint. Method shouldn't be called for anything what is not a string.

Reply:
I was being over-protective, in case somebody attempts to use the library elsewhere. :smile: